standalone-metastore/metastore-rest-catalog/pom.xml

```diff
@@ -42,6 +42,12 @@
       <artifactId>hive-iceberg-catalog</artifactId>
       <version>${hive.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.hive</groupId>
+      <artifactId>hive-iceberg-handler</artifactId>
```

Comment:
why do we need `iceberg-handler` in `metastore-rest-catalog`?

Reply:
Done